src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,813 @@
+        }
+    }
+
+    private void verifyExportTime(long endDateTimeEpoch, boolean ignoreLastExportDateTime, boolean exportAll) {
+        if (!ignoreLastExportDateTime) {
+            if (!exportAll) {
+                Item lastExportDateTime = ddbExportTimeTable.getItem("studyId", studyId);
+                long lastExportDateTimeEpoch = lastExportDateTime.getLong("lastExportDateTime");
+                assertNotNull(lastExportDateTimeEpoch);
+
+                // the time recorded in export time table should be equal to the date time we submit to the export request
+                assertEquals(endDateTimeEpoch, lastExportDateTimeEpoch);
+            } else {
+                Iterable<Item> scanOutcomes = ddbScanHelper.scan(ddbExportTimeTable);
+
+                // verify if all studies' last export date time are modified
+                for (Item item: scanOutcomes) {
+                    long lastExportDateTimeEpoch = item.getLong("lastExportDateTime");
+                    assertEquals(endDateTimeEpoch, lastExportDateTimeEpoch);
+                }
+            }
+        } else {
+            if (!exportAll) {
+                Item lastExportDateTime = ddbExportTimeTable.getItem("studyId", studyId);
+                Long lastExportDateTimeEpoch = lastExportDateTime.getLong("lastExportDateTime");
+                assertNotNull(lastExportDateTimeEpoch);
+
+                assertTrue(lastExportDateTimeEpoch < endDateTimeEpoch);
+            } else {
+                Iterable<Item> scanOutcomes = ddbScanHelper.scan(ddbExportTimeTable);
+
+                // verify if all studies' last export date time are modified
+                for (Item item: scanOutcomes) {
+                    Long lastExportDateTimeEpoch = item.getLong("lastExportDateTime");
+                    assertTrue(lastExportDateTimeEpoch < endDateTimeEpoch);
+                }
+            }
+        }
+    }
+
+    private void assertExport(BridgeExporterRequest exporterRequestOri, ExportType exportType, String uploadId, int offset, boolean exportAll, boolean ignoreLastExportDateTime, boolean noUpload) throws IOException, InterruptedException, SynapseException {
+        DateTime dateTimeBeforeExport;
+        BridgeExporterRequest exporterRequest;
+
+        if (exporterRequestOri == null) {
+            dateTimeBeforeExport = DateTime.now();
+            BridgeExporterRequest.Builder exporterRequestBuilder;
+            if (!exportAll) {
+                exporterRequestBuilder = new BridgeExporterRequest.Builder().withEndDateTime(dateTimeBeforeExport).withExportType(
+                        exportType).withStudyWhitelist(ImmutableSet.of(studyId)).withTag("ex integ test");
+            } else {
+                exporterRequestBuilder = new BridgeExporterRequest.Builder().withEndDateTime(dateTimeBeforeExport).withExportType(
+                        exportType).withTag("ex integ test export all");
```

Comment:
i put the isProd verification to some specific tests instead of this place. is that ok?